cli/config/compose/profiles/fleet/docker-compose.yml

```diff
@@ -29,7 +29,7 @@ services:
       test: "curl -f http://localhost:5601/login | grep kbn-injected-metadata 2>&1 >/dev/null"
       retries: 600
       interval: 1s
-    image: "docker.elastic.co/${kibanaDockerNamespace:-beats}/kibana:${kibanaVersion:-8.0.0-SNAPSHOT}"
```

Comment:
I always wondered about this one :)

Reply:
Wrong copy&paste on my side for sure 🤦 

I think I'm gonna merge it after the CI passes